provider/win_dump_utils.py

```diff
@@ -128,9 +128,11 @@ def dump_windbg_check(test, params, session):
     log_file = params["dump_analyze_file"]
     chk_dump_cmd = utils_misc.set_winutils_letter(session,
                                                   chk_dump_cmd)
-    session.cmd(chk_dump_cmd)
+    status, output = session.cmd_status_output(chk_dump_cmd)
```

Comment:
The output is not used.We can add it to test fail log.

Reply:
Sure,thanks